README.md

```diff
@@ -17,3 +17,79 @@ New serialization approaches should be in their own directories with an appropri
 We will do a minimum review of new proposals before merging.
 
 Suggestions to existing serialization approaches sill be reviewed by the contributors of the original serializations approach before merging.
+
+## Operating Principles
+1. The [logical model](https://github.com/spdx/spdx-3-model/tree/main/model) is the single authoritative source for SPDXv3 content.
+All examples submitted to the playground should correspond to the given model examples.
+2. Each serialization method has a sponsor that is responsible for creating the examples and test code for that method.
+3. The barrier to entry should be minimal. A sponsor may create as many examples as they deem appropriate for defining the method.
+4. Although examples may be initially submitted to illustrate ideas before code has been developed to process them, it will
+eventually be necessary to demonstrate that the examples correctly reflect the model.
+The code for a method includes four programs:
```

Comment:
I'm thinking we may not just skip the details for the programs - overall, this may be a rather intimidating list to submit ideas (e.g. delete lines 28-32)